tests/integration/test_system_ddl_worker_queue/test.py

```diff
@@ -68,3 +80,75 @@ def test_distributed_ddl_queue(started_cluster):
             )
             == "ok\n"
         )
+
+    node1.query(
+        "ALTER TABLE testdb.test_table ON CLUSTER test_cluster DROP COLUMN somecolumn",
+        settings={"replication_alter_partitions_sync": "2"},
+    )
```

Comment:
I would suggest to use different tables for different test runs, otherwise it can be flaky (i.e. if some query before this  fails it won't be executed, we can use `finally`, but separate tables looks better)